PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutClient.kt

```diff
@@ -143,4 +219,12 @@ class PayPalWebCheckoutClient internal constructor(
         analyticsService.sendAnalyticsEvent("paypal-web-payments:succeeded", orderId)
         listener?.onPayPalWebSuccess(result)
     }
+
+    fun vault(activity: AppCompatActivity, setupTokenId: String, approveVaultHref: String) {
```

Comment:
The name `vault()` doesn't have to be final here if we can find a better name for starting the vault flow. Also wondering if `setupTokenId` and `approveVaultHref` should be placed in a `PayPalWebCheckoutVaultRequest` parameter object.